akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/Effect.scala

```diff
@@ -230,6 +233,64 @@ object Effect {
 
   final case class TimerCancelled(key: Any) extends Effect
 
+  /**
+   * The behavior initiated an ask of a targeted recipient.
```

Comment:
Perhaps good to spell it out that it is about `ActorContext.ask` since there also is the Patterns-ask that I think we will not be able to make an effect of.